build_tools/cmake/ctest_all.sh

```diff
@@ -124,6 +124,8 @@ if [[ "${OSTYPE}" =~ ^msys ]]; then
     # TODO(#11070): Fix argument/result signature mismatch
     "iree/tests/e2e/tosa_ops/check_vmvx_local-sync_microkernels_fully_connected.mlir"
     "iree/tests/e2e/tosa_ops/check_vmvx_local-sync_microkernels_matmul.mlir"
+    # Flaky on CI for unknown reasons, skip.
+    "iree/tools/test/iree-dump-parameters.txt.test"
```

Comment:
https://github.com/iree-org/iree/actions/runs/13217499076/job/36898486652#step:10:3413

```

The following tests FAILED:
	 12 - iree/tools/test/iree-dump-parameters.txt.test (Failed) driver=local-task hostonly iree/tools/test test-type=lit-test
	 23 - iree/tools/test/parameters_unscoped.mlir.test (Failed) driver=local-task hostonly iree/tools/test test-type=lit-test
```

can skip that other test too:
```suggestion
    # Flaky on CI for unknown reasons, skip.
    "iree/tools/test/iree-dump-parameters.txt.test"
    "iree/tools/test/parameters_unscoped.mlir.test"
```